tests/conftest.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+import pytest
+
+
+@pytest.fixture()
+def test_inputs_path():
```

Comment:
This is only used in one place. As it's not shared across test modules, I'd be tempted to move this fixture into the file it's used in.

Reply:
Fair point!